lib/xhr.js

```diff
@@ -337,7 +340,11 @@ function enableXHROverH2(XMLHttpRequest, configuration) {
 
 
                 var transport = configuration.getTransport(proxyTransportUrl);
-
+                var wsErrorHandLer = function (/*e*/) {
+                    if (self.readyState === XMLHttpRequest.UNSENT && options.accelerationStrategy === 'connected') {
+                        self.send();
+                    }
+                };
```

Comment:
Can this function be created outside of the current scope?  Otherwise, a new one will be created every time this method is called.  Otherwise, I'm not clear why it was moved into a variable?